src/prepared_statement.rs

```diff
@@ -0,0 +1,174 @@
+
+use frame::{Frame, Flag};
+use IntoBytes;
+use frame::parser::parse_frame;
+use authenticators::Authenticator;
+use error;
+use transport::CDRSTransport;
+use consistency::Consistency;
+use client::Session;
+use query::QueryParamsBuilder;
+use types::value::Value;
+use frame::frame_response::ResponseBody;
+use std::collections::BTreeMap;
+use std::convert::Into;
+use byteorder::{BigEndian, WriteBytesExt};
+
+#[derive(Debug)]
+pub struct PreparedStatement {
```

Comment:
It seems to be duplicating [this](https://github.com/AlexPikalov/cdrs/blob/master/examples/prepare_execute.rs#L35) functionality.

The other think is that this approach does not allow to prepare some query just once and then execute it few times with different values on different stages of application run-cycle. For instance, prepare inserting some row on application init stage and insert different rows each time we get related request.